moveit_core/utils/include/moveit/utils/moveit_error_code.h

```diff
@@ -0,0 +1,76 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2021
```

Comment:
I know this is nitpicky, but you have to specify the holder of the copyright. That's either you (if you worked on this privately) or more likely the company that paid you for working on this.

Reply:
I wasn't sure what to use that's why I left it empty, will update it right now